addons/event_sale/models/event.py

```diff
@@ -249,7 +249,7 @@ def summary(self):
         order = self.sale_order_id.sudo()
         order_line = self.sale_order_line_id.sudo()
         if not order or float_is_zero(order_line.price_total, precision_digits=order.currency_id.rounding):
-            payment_status = _('Free')
+            payment_status = _('Free admission')
```

Comment:
I thought we didn't allow changing terms in a stable?

Reply:
Yeah, but it's specific case. Check commit description and discussions

Reply:
solved